app/controllers/admin/delegates_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Admin
+  class DelegatesController < ApplicationController
+    before_action :require_svie_admin
+
+    def index
+      groups = Group.order(:name)
+      @groups = groups.select { |group| group.delegate_count.positive? }
```

Comment:
How about `@groups_witht_delegates, @groups_without_delegates = groups.partition ...`?

Reply:
Thanks, it is a much nicer solution.